common/cache/src/cache/lru.rs

```diff
@@ -695,3 +696,13 @@ impl<'a, K, V> ExactSizeIterator for IterMut<'a, K, V> {
         self.0.len()
     }
 }
+
+// count bytes for data accessors
+impl<K, T> Meter<K, Vec<T>> for Vec<u8> {
```

Comment:
This bytesmeter should be placed in the `meter` mod.

Reply:
u are right.. will fix it.